index.d.ts

```diff
@@ -55,6 +66,11 @@ export type DailyMeetingState =
   | 'left-meeting'
   | 'error';
 
+export interface DailyParticipantsObject {
```

Comment:
From my (limited) understanding of `Record`, it seems like it's mostly useful for succinctly limiting the keys in an object to a specific set of values, but isn't the most common/idiomatic. My inclination was to stick to using the indexable values like before.

Reply:
Also, I updated `type`s to be `interface`s where possible, just for consistency with how we've been doing things in this file.

Reply:
Yeah, that's totally fine -- these changes are equivalent and it's largely up to your code preferences so up to you! FWIW `Record` is actually very commonly used these days as a shorthand (I saw that stack overflow where they're saying it's not as idiomatic), but I'm not sure it even helps readability in this case, my bad 😅

Reply:
Haha you probably know better than me what's going on "these days". We'll keep it like this for now, but I'll keep an open mind going forward.